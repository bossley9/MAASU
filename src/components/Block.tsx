import React, { FC, ReactNode } from 'react'

type Props = {
  children: ReactNode
  className?: string
}

export const Block: FC<Props> = ({ children, className = '' }) => {
  return <div className={`my4 ${className}`}>{children}</div>
}
