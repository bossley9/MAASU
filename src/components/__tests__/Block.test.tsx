import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import { Block } from '../Block'

describe('render', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Block>hello</Block>, div)
    ReactDOM.unmountComponentAtNode(div)
  })
})

it('renders a matching snapshot', () => {
  const test = renderer.create(<Block>hello</Block>)
  expect(test.toJSON()).toMatchSnapshot()
})
