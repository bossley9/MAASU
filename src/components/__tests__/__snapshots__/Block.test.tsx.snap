// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a matching snapshot 1`] = `
<div
  className="my4 "
>
  hello
</div>
`;
